packages/apps/human-app/frontend/src/modules/signup/worker/views/sign-up-worker.page.tsx

```diff
@@ -48,7 +48,7 @@ export function SignUpWorkerPage() {
       alert={
         isError ? (
           <Alert color="error" severity="error" sx={{ width: '100%' }}>
-            {getErrorMessageForError(isError, handleSignupError)}
+            {getErrorMessageForError(error, handleSignupError)}
```

Comment:
maybe it worth to narrow down the type of the first argument of `getErrorMessageForError `.
wondering if its type can just be `Error`

Reply:
@KirillKirill nice point!
Applied in https://github.com/humanprotocol/human-protocol/pull/3277/commits/f732d29432e226e4d687b41a79c7748e85d9eea1